src/AutoBuild.jl

```diff
@@ -1079,7 +1083,7 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
                         end
                     end
                     if adjust_LIBPATH
-                        if !isempty(get(ENV, LIBPATH_env, ""))
+                        if !isempty(get(ENV, LIBPATH_env, expanduser(LIBPATH_default)))
                             env_mapping[LIBPATH_env] = string(LIBPATH, $(repr(pathsep)), ENV[LIBPATH_env])
```

Comment:
```suggestion
                            env_mapping[LIBPATH_env] = string(LIBPATH, $(repr(pathsep)), get(ENV, LIBPATH_env, expanduser(LIBPATH_default)))
```

Reply:
@staticfloat if you update your branch with this, I can try it again.

Reply:
You can try now

Reply:
Success!
https://travis-ci.com/github/simonbyrne/mpichtest/jobs/328408776#L631-L632